tasks.py

```diff
@@ -136,7 +136,7 @@ def _detect_space(branch=None, yes=False):
 
 DEPLOY_RULES = (
     ('prod', _detect_prod),
-    ('stage', lambda _, branch: branch == 'master'),
+    ('stage', lambda _, branch: branch == 'release'),
```

Comment:
Here's what confuses me about using git-flow with continuous deployment:
* *Every* update to master is tagged, either via hotfix or release
* If devs push features and hotfixes to their forks and not the main repo, Travis never builds those branches with `TRAVIS_PULL_REQUEST = 'false'`

One solution would be to have devs push feature and hotfix branches to origin and not to use forks. This is a little inefficient, since it triggers two Travis builds (one for the push to 18f/hotfix/<...>, and another for the pull request to 18f/master), but builds are free. Also, I think this is what everyone on the team except for me is already doing, so it wouldn't require anyone else to change workflows. Is this compatible with (or the same as) what you're proposing @arowla?